services/auth/token/helpers/generateIAMPolicy.js

```diff
@@ -0,0 +1,21 @@
+// Helper function to generate an IAM policy
+function generateIAMPolicy(principalId, effect, resource) {
+  const authResponse = {};
+
+  authResponse.principalId = principalId;
+  if (effect && resource) {
+    const policyDocument = {};
+    policyDocument.Version = '2012-10-17';
+    policyDocument.Statement = [];
+    const statementOne = {};
+    statementOne.Action = 'execute-api:Invoke';
+    statementOne.Effect = effect;
+    statementOne.Resource = resource;
+    policyDocument.Statement[0] = statementOne;
+    authResponse.policyDocument = policyDocument;
+  }
+
+  return authResponse;
+}
+
+export default generateIAMPolicy;
```

Comment:
you are exporting the function name in file but not in other files? Should aim for consistently.

Reply:
I hear you and the changes has been made. Typical component exports is done like this, so probably did this from react experience :P